spec/factories/listings.rb

```diff
@@ -6,5 +6,6 @@
     description { 'Gated park with night lamp' }
     price { 200 }
     address {'Lövåsvägen 21 16733 Bromma'}
+    association :landlord, factory: :user
```

Comment:
What is the landlord association?

Reply:
each listing should have a landlord

Reply:
But we should not have a specific landlord role, it should be a user or a subscriber